specification/sql/resource-manager/readme.md

```diff
@@ -495,6 +495,7 @@ input-file:
 - ./Microsoft.Sql/preview/2020-02-02-preview/ServerSecurityAlertPolicies.json
 - ./Microsoft.Sql/preview/2020-02-02-preview/operations.json
 - ./Microsoft.Sql/preview/2020-02-02-preview/managedDatabases.json
+- ./Microsoft.Sql/preview/2020-02-02-preview/managedInstances.json
```

Comment:
It seems the `2020-02-02-preview` hasn't been added to SDK generation in `readme.*.md`. Please add specific language for sdk generation.

Reply:
Hi @ChenTanyi . I tried to do this, but I am not sure what needs to be changed. We need .Net SDK generation and Python SDK generation for PS and CLI clients. For the path "specification/sql/resource-manager/" I don't see any readme.csharp.md or readme.python.md files, and all api versions from all time are here. Can you help me pointing out in more details what is missing? Thanks

Reply:
csharp is https://github.com/Azure/azure-rest-api-specs/tree/3336f84f2e03b2f201ccdae0ceeb4a2d087354d9/specification/sql/resource-manager#c
python is https://github.com/Azure/azure-rest-api-specs/tree/3336f84f2e03b2f201ccdae0ceeb4a2d087354d9/specification/sql/resource-manager#python

It seems both of they don't need specific version in readme. If you only need these two SDK, maybe it no need to change.